### ISSUE TITLE
Can't reproduce the results on AffectNet-7 and AffectNet-8

### ISSUE THREAD

**Issue by zerooooone** [Talented-Q/POSTER_V2#14]
Thanks for your great work!
I tested the pre-trained model trained on AffectNet you provided and got similar results(fig 1&2). But when I train with your code, I only got poor training accuracy curve(fig 3). Is there any training detail or trick I ignored? How can I reach your accuracy?
![image](https://github.com/Talented-Q/POSTER_V2/assets/48344537/d798a378-75ec-44fb-9fa4-85313c4ddbe4)
![image](https://github.com/Talented-Q/POSTER_V2/assets/48344537/4c5a4cf6-48ca-4d1d-a9db-c744389f0d03)
![image](https://github.com/Talented-Q/POSTER_V2/assets/48344537/ffed8a97-292f-4ac2-97a4-1f257eb422c8)

**Comment by Talented-Q** [comment]
![image](https://github.com/Talented-Q/POSTER_V2/assets/79841528/edbe9e76-f406-42fb-963b-52da555accec)

Thank you for your interest in this paper. sampler=ImbalancedDatasetSampler(train_dataset)。
The Affectnet dataset should be oversampled during training. sampler=ImbalancedDatasetSampler(train_dataset) should be added to train_loader. We apologize that we did not point out that we had given you the wrong impression. We will make changes to the above issues in the coming work.
